saleor/graphql/product/types/products.py

```diff
@@ -192,13 +192,8 @@ class ProductVariant(CountableDjangoObjectType):
             "This field will be removed after 2020-07-31."
         ),
     )
-    price_override = graphene.Field(
-        Money,
-        description=(
-            "Override the base price of a product if necessary. A value of `null` "
-            "indicates that the default product price is used."
-        ),
-    )
+    price = graphene.Field(Money, description=("Base price of a product variant."),)
```

Comment:
I would extend this description with sth like: "This field is restricted for admins. Use the `pricing` field to get the public price for customers." It will be less confusing for API users when they see that the regular price field is protected with permissions.